packages/blockchain-wallet-v4-frontend/src/data/components/brokerage/types.ts

```diff
@@ -25,9 +25,22 @@ export type BankStatusType =
   | 'BANK_TRANSFER_ACCOUNT_NAME_MISMATCH'
   | 'DEFAULT_ERROR'
 
+export enum BankDepositStepType {
+  ADD_BANK = 'ADD_BANK',
+  BANK_LIST = 'BANK_LIST',
+  CONFIRM = 'CONFIRM',
+  DEPOSIT_METHODS = 'DEPOSIT_METHODS',
+  DEPOSIT_STATUS = 'DEPOSIT_STATUS',
+  ENTER_AMOUNT = 'ENTER_AMOUNT'
+}
+
 export type BrokerageStepPayload =
   | {
-      step: AddBankStepType.ADD_BANK | AddBankStepType.ADD_BANK_HANDLER
+      step:
+        | AddBankStepType.ADD_BANK
+        | AddBankStepType.ADD_BANK_HANDLER
+        | BankDepositStepType.DEPOSIT_METHODS
+        | BankDepositStepType.ADD_BANK
```

Comment:
would  be good to rename one `ADD_BANK` we have same type prop on two different types `AddBankStepType` and `BankDepositStepType`